src/libcommonserver/io/iohelper.cpp

```diff
@@ -284,8 +285,12 @@ bool IoHelper::getItemType(const SyncPath &path, ItemType &itemType) noexcept {
     const bool isSymlink = _isSymlink(path, ec);
 
     itemType.ioError = stdError2ioError(ec);
+#ifndef WIN32
```

Comment:
You did not use the same macro as below (WIN32 vs _WIN32). Is there a particular reason£? If not, it would be great to uniformize.
Also avoid, when it is possible, the negation. Use 'ifdef' instead on 'ifndef'

Reply:
No particular reason, here are the corrections:
- For _WIN32 [ff0ce59](https://github.com/Infomaniak/desktop-kDrive/pull/338/commits/ff0ce5938bce3875ef902d238be29ca77bf5b8a8) 
- Use of ifdef instead of ifndef [d120287](https://github.com/Infomaniak/desktop-kDrive/pull/338/commits/d12028727708e33c9975d589839df07ae9ccc015)